pkg/odo/cli/config/view.go

```diff
@@ -26,10 +23,9 @@ var viewExample = ktemplates.Examples(`# For viewing the current configuration f
 
 // ViewOptions encapsulates the options for the command
 type ViewOptions struct {
-	contextDir  string
-	devfilePath string
-	devfileObj  parser.DevfileObj
-	IsDevfile   bool
+	contextDir string
+	*genericclioptions.Context
+	//*clicomponent.PushOptions
```

Comment:
You can remove this commented code

Reply:
Thanks, Philippe!
Removed the commented code :)